recipes/gdal/post_3.5.0/patches/3.5.2/0-replace-find-package.patch

```diff
@@ -0,0 +1,266 @@
+diff --git a/alg/CMakeLists.txt b/alg/CMakeLists.txt
```

Comment:
Good catch! Yes, please, re-use same file

Reply:
If I try to re-use the same file, I get:
```
gdal/3.5.2: WARN: /home/jpilet/.conan/data/gdal/3.5.2/_/_/build/75d06ef31f24be9258c8c7f1bc3696de61c66b4f/patches/3.5.1-2/0-replace-find-package.patch: file 3/7:         b'cmake/helpers/CheckDependentLibraries.cmake'
gdal/3.5.2: WARN: /home/jpilet/.conan/data/gdal/3.5.2/_/_/build/75d06ef31f24be9258c8c7f1bc3696de61c66b4f/patches/3.5.1-2/0-replace-find-package.patch:  hunk no.3 doesn't match source file at line 321
gdal/3.5.2: WARN: /home/jpilet/.conan/data/gdal/3.5.2/_/_/build/75d06ef31f24be9258c8c7f1bc3696de61c66b4f/patches/3.5.1-2/0-replace-find-package.patch:   expected: b'endif ()'
gdal/3.5.2: WARN: /home/jpilet/.conan/data/gdal/3.5.2/_/_/build/75d06ef31f24be9258c8c7f1bc3696de61c66b4f/patches/3.5.1-2/0-replace-find-package.patch:   actual  : b'endif()'
gdal/3.5.2: WARN: /home/jpilet/.conan/data/gdal/3.5.2/_/_/build/75d06ef31f24be9258c8c7f1bc3696de61c66b4f/patches/3.5.1-2/0-replace-find-package.patch: file 3/7:         b'cmake/helpers/CheckDependentLibraries.cmake'
gdal/3.5.2: WARN: /home/jpilet/.conan/data/gdal/3.5.2/_/_/build/75d06ef31f24be9258c8c7f1bc3696de61c66b4f/patches/3.5.1-2/0-replace-find-package.patch:  hunk no.4 doesn't match source file at line 379
gdal/3.5.2: WARN: /home/jpilet/.conan/data/gdal/3.5.2/_/_/build/75d06ef31f24be9258c8c7f1bc3696de61c66b4f/patches/3.5.1-2/0-replace-find-package.patch:   expected: b'  TARGETS json-c::json-c JSONC::JSONC'
gdal/3.5.2: WARN: /home/jpilet/.conan/data/gdal/3.5.2/_/_/build/75d06ef31f24be9258c8c7f1bc3696de61c66b4f/patches/3.5.1-2/0-replace-find-package.patch:   actual  : b''
gdal/3.5.2: WARN: /home/jpilet/.conan/data/gdal/3.5.2/_/_/build/75d06ef31f24be9258c8c7f1bc3696de61c66b4f/patches/3.5.1-2/0-replace-find-package.patch: file 3/7:         b'cmake/helpers/CheckDependentLibraries.cmake'
gdal/3.5.2: WARN: /home/jpilet/.conan/data/gdal/3.5.2/_/_/build/75d06ef31f24be9258c8c7f1bc3696de61c66b4f/patches/3.5.1-2/0-replace-find-package.patch:  hunk no.5 doesn't match source file at line 482
gdal/3.5.2: WARN: /home/jpilet/.conan/data/gdal/3.5.2/_/_/build/75d06ef31f24be9258c8c7f1bc3696de61c66b4f/patches/3.5.1-2/0-replace-find-package.patch:   expected: b'endif ()'
gdal/3.5.2: WARN: /home/jpilet/.conan/data/gdal/3.5.2/_/_/build/75d06ef31f24be9258c8c7f1bc3696de61c66b4f/patches/3.5.1-2/0-replace-find-package.patch:   actual  : b'  if (NOT DEFINED SQLite3_HAS_COLUMN_METADATA)'
gdal/3.5.2: /home/jpilet/.conan/data/gdal/3.5.2/_/_/build/75d06ef31f24be9258c8c7f1bc3696de61c66b4f/patches/3.5.1-2/0-replace-find-package.patch: source file is different - b'cmake/helpers/CheckDependentLibraries.cmake'
gdal/3.5.2: WARN: /home/jpilet/.conan/data/gdal/3.5.2/_/_/build/75d06ef31f24be9258c8c7f1bc3696de61c66b4f/patches/3.5.1-2/0-replace-find-package.patch: file 5/7:         b'gdal.cmake'
gdal/3.5.2: WARN: /home/jpilet/.conan/data/gdal/3.5.2/_/_/build/75d06ef31f24be9258c8c7f1bc3696de61c66b4f/patches/3.5.1-2/0-replace-find-package.patch:  hunk no.1 doesn't match source file at line 795
gdal/3.5.2: WARN: /home/jpilet/.conan/data/gdal/3.5.2/_/_/build/75d06ef31f24be9258c8c7f1bc3696de61c66b4f/patches/3.5.1-2/0-replace-find-package.patch:   expected: b'                 ${CMAKE_CURRENT_BINARY_DIR}/GDALConfig.cmake @ONLY)'
gdal/3.5.2: WARN: /home/jpilet/.conan/data/gdal/3.5.2/_/_/build/75d06ef31f24be9258c8c7f1bc3696de61c66b4f/patches/3.5.1-2/0-replace-find-package.patch:   actual  : b'    GLOBAL_PROPERTY'
gdal/3.5.2: /home/jpilet/.conan/data/gdal/3.5.2/_/_/build/75d06ef31f24be9258c8c7f1bc3696de61c66b4f/patches/3.5.1-2/0-replace-find-package.patch: source file is different - b'gdal.cmake'

```

because the offsets do not match. The same changes are applied, but line positions changed.
I do not know how to make a patch file that is compatible with 2 different source files.

Reply:
RIP! Okay that's fine then